dts/bindings/adc/nxp,lpc-lpadc.yaml

```diff
@@ -75,6 +75,10 @@ properties:
     type: phandles
     description: References to required regulators which must be enabled for LPADC to function
 
+  nxp,reference-supply-value:
```

Comment:
I don't think we need to add this property- instead, we can read the `zephyr,vref-mv` property, and attempt to set the regulator to that voltage

Reply:
yes, the property zephyr,vref-mv can do this